opre_ops/ops_site/admin.py

```diff
@@ -0,0 +1,8 @@
+
+from django.contrib import admin
+from ops_site.models import Agency
+
+
+@admin.register(Agency)
+class AgencyAdmin(admin.ModelAdmin):
+    list_display = ("name",)
```

Comment:
[Suggestion] Since you already seeded the `nickname` data, you may want to add them in the display for viewing purposes

Reply:
I can do that!